src/main/java/com/salesforce/mirus/offsets/MirusOffsetTool.java

```diff
@@ -63,6 +64,9 @@ public static void main(String[] argv) throws IOException {
     if (args.resetOffsets && (args.fromFile == null || args.fromFile.isEmpty())) {
       throw new ParameterException("--reset-offsets requires --from-file to be set");
     }
+    if (args.writeNullOffsets && !args.describe) {
```

Comment:
Not sure of the naming here - "write" implies changing something in storage, so writing to the offsets topic in this case.  Should be "showNullOffsets"?

Reply:
True that, I had show then changed it. I will change it back to _show_